prisma/schema.prisma

```diff
@@ -11,14 +14,83 @@ datasource db {
   directUrl = env("DATABASE_DIRECT_URL")
 }
 
+enum Gender {
+  MALE
+  FEMALE
+}
+
+enum FormatOfWork {
+  OFFLINE
+  ONLINE
+  BOTH
+}
+
 model Specialist {
-  id        String @id @default(uuid())
-  gender    String
-  firstName String
-  lastName  String
+  id                String           @id @default(uuid()) @db.Uuid
+  createdAt         DateTime         @default(now())
+  firstName         String           @db.VarChar(64)
+  lastName          String           @db.VarChar(64)
+  surname           String?          @db.VarChar(64)
+  specializations   Specialization[]
+  gender            Gender
+  yearsOfExperience Int?             @db.SmallInt
+  formatOfWork      FormatOfWork
+  placesOfWork      PlaceOfWork[]
+  therapies         Therapy[]
+  isFreeReception   Boolean
+  description       String           @db.Text
+  phone             String?          @db.VarChar(15)
```

Comment:
email should probably have a @unique constraint

Reply:
since this is the contacts section, I think it's reasonable to assume that different specialists can be reached out to by the same email

Reply:
Makes sense. Thanks!